### PR TITLE
remove internal used session restore() function

### PR DESCRIPTION
**Pull request by Uzlopak**
@rclmenezes 

any thoughts on this?
It makes it obvious that rolling is basically not implemented I guess.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
@rclmenezes 
Let us merge this before you can implement fix for #126

**Review by mcollina — APPROVED** [review]
lgtm

File: lib/fastifySession.js [hunk 1]
```diff
@@ -78,25 +78,14 @@ function session (fastify, options, next) {
         })
         return
       }
-      if (options.rolling) {
```
Comment (rclmenezes) [inline lib/fastifySession.js:81]: Btw, we didn't remove `rolling` from the docs.

This is how `express-session` uses rolling:
https://github.com/expressjs/session/blob/1010fadc2f071ddf2add94235d72224cf65159c6/index.js#L470-L472

Comment (Uzlopak) [reply lib/fastifySession.js:81]: I know, we have to reimplement or check if rolling is working as expected.

Comment (rclmenezes) [reply lib/fastifySession.js:81]: Cool I made an issue: https://github.com/fastify/session/issues/148

**Pull request merged**